01_fib/ailyntang/fib.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func fib(n int) {
+	if n < 1 || n > 92 {
+		fmt.Fprintln(out, "fib.go only handles integers between 1 to 92 inclusive")
+		return
+	}
+
+	series := genFib(n)
```

Comment:
I'd suggest removing the `series` variable. It is used only once, and giving it a name doesn't add any more clarity. So something like:
```
for _, num := range genFib(n) {
        fmt.Fprintln(out, num)
}
```

is just as readable and more compact.

No need to make this change though. Just something to think about for later labs. You can even choose to ignore it if you prefer it your way.

Reply:
This makes sense, thanks for the suggestion.